src/api/lib/storage/backends/consul.js

```diff
@@ -0,0 +1,109 @@
+var Storage = require('../');
+var consul = require('consul');
+var path = require('path');
+
+module.exports = ConsulBackend;
+
+function ConsulBackend(options) {
+	options = options || {};
+	this._prefix = options.prefix || 'billimanjaro';
```

Comment:
This function confused me a bit but I think it's fine. I'll re-read it a bit more.